install/dependencies.md

```diff
@@ -18,55 +18,93 @@ Use `yum install <package>` or the graphical "Add/Remove Software" program.
+*   **mesa-libGL-devel:** for /usr/include/GL/gl.h and for /usr/lib[64]/libGL.so
+*   **mesa-libGLU-devel:** for /usr/include/GL/glu.h and for /usr/lib[64]/libGLU.so
 *   **mysql-devel:** for /usr/bin/mysql_config
 *   **fftw-devel:** for /usr/include/fftw3.h and for /usr/lib/libfftw3.so
-*   **cfitsio-devel:** for /usr/include/fitsio2.h and for /usr/lib/libcfitsio.so
-*   **graphviz-devel:** for /usr/include/graphviz/gvc.h and for /usr/lib/libgvc.so
-*   **avahi-compat-libdns_sd-devel:** for /usr/include/dns_sd.h and for /usr/lib/libdns_sd.so
+*   **libuuid-devel** required during the build. More [info here](https://root-forum.cern.ch/t/cannot-compile-root-v6-22-08-with-debug-symbols-and-all-options/44242).
 *   **openldap-devel:** for /usr/include/ldap.h and for /usr/lib/libldap.so
-*   **python-devel:** for /usr/include/python2.7/Python.h and for /usr/lib/libpython2.7.so
-*   **python-numpy-devel:** needed for PyMVA
+*   **python-devel:** (only on RHEL 7 based systems with default Python v.2) for /usr/include/python2.7/Python.h and for /usr/lib/libpython2.7.so
+*   **python3-numpy:** (**numpy** on RHEL 7 based systems) needed for PyMVA
 *   **libxml2-devel:** for /usr/bin/xml2-config
 *   **gsl-devel:** for /usr/include/gsl/gsl_version.h and for /usr/lib/libgsl.a
-*   **r-base:** needed for R bindings. In addition R packages **Rcpp** and **RInside** need to be installed.
+*   **readline-devel:** required at the linking stage.
+*   RHEL based distributions (CentOS, Scientific Linux) will require enabling of the [EPEL](https://fedoraproject.org/wiki/EPEL) additional software repository and in order to install packages below:
+*   **ftgl-devel:** for /usr/bin/pkg-config
+*   **cfitsio-devel:** for /usr/include/fitsio2.h and for /usr/lib/libcfitsio.so
+*   **R-devel, R-Rcpp-devel, R-RInside-devel:** needed for R bindings. 
+*   RHEL 8 based distributions (CentOS 8) will require enabling of the PowerTools](https://wiki.centos.org/FAQ/CentOS8/UnshippedPackages) additional software repository and in order to install packages below:
+*   **avahi-compat-libdns_sd-devel:** for /usr/include/dns_sd.h and for /usr/lib/libdns_sd.so
+*   **glew-devel:** for /usr/include/GL/glew.h and for /usr/lib[64]/libGLEW.so
+*   **graphviz-devel:** for /usr/include/graphviz/gvc.h and for /usr/lib/libgvc.so
 
-As a one-liner:
+As a one-liner for Fedora 33:
+```bash
+sudo yum install redhat-lsb-core gcc-gfortran pcre-devel \
+mesa-libGL-devel mesa-libGLU-devel glew-devel ftgl-devel mysql-devel \
+fftw-devel cfitsio-devel graphviz-devel libuuid-devel \
+avahi-compat-libdns_sd-devel openldap-devel python3-numpy \
+libxml2-devel gsl-devel readline-devel R-devel R-Rcpp-devel R-RInside-devel
+```
+
+CentOS 8 requores adding EPEL repository. Additionally, some packages in CentOS 8 [were moved to the PowerTools repository](https://root-forum.cern.ch/t/root-dependencies-can-not-install-in-centos8/42010). Therefore it needs to be enabled as well:
```

Comment:
```suggestion
CentOS 8 requires adding the EPEL repository. Additionally, some packages in CentOS 8 [were moved to the PowerTools repository](https://root-forum.cern.ch/t/root-dependencies-can-not-install-in-centos8/42010). Therefore it needs to be enabled as well:
```

Reply:
My only concern is following. I simply compiled ROOT with `-Dall=ON` and `-DCMAKE_BUILD_TYPE=DEBUG`. However I am not sure if above options require the largest possible number of the dependencies. Please let me know.

Reply:
can you add a commit with this and the other minor fixes please?

Reply:
done!